### ISSUE TITLE
Address fields are USA only

### ISSUE THREAD

**Issue by dmgeurts** [opennetadmin/ona#153]
In the UK we don't have ZIP codes nor states, but postcodes, counties and countries...

**Comment by mattpascoe** [comment]
yes.. I've had issue #19 for this for awhile.. obviously it has not been high on my list.. It does need attention.
